app/views/groups/members/_updated_at.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= render UpdatedComponent.new(
+  description: t(:"groups.members.index.updated"),
+  updated_at: distance_of_time_in_words(Time.now, member.updated_at)
```

Comment:
**thought (blocking):** Wouldn't it make more sense to put the `distance_of_time_in_words` into your component, that way it does not need to be setup every time you use the component?

Reply:
I think that would make too much sense :) Updated in 42c31e6fbb8529ab2a3379eeb513c91043cf1e12